docs/service-guide/create-service.rst

```diff
@@ -32,72 +32,16 @@ If you are using the FastAPI template, tagging in this fashion is required since
 Create the Docker image
 =======================
 
-The Docker image can be stored in any container registry that is usable by Kubernetes, but for Rubin-developed services, we normally use DockerHub.
-(We may switch to the Google Container Registry later, but for now DockerHub is used for all images.)
+The Docker image can be stored in any container registry that is usable by Kubernetes, but for Rubin-developed services using the FastAPI template, we usually push both to the `GitHub Container Registry <https://ghcr.io>`__ and Docker Hub.  Google Artifact Registry is in play for Science Platform images and may eventually be used more widely.  We may eventually stop publishing to Docker Hub; our workflow is centered on GitHub and the long-term future of Docker-the-company does not look very secure.
 If your image must be stored in a private container registery, the credentials for that registry must be added to the pull secret.
 
 If you use the FastAPI service template, a ``Dockerfile`` will be created as part of the new repository template, and GitHub Actions will be set up in the new repository to build and push new Docker images for tagged releases.
-To enable this workflow, you must create two secrets in your new GitHub repository, ``DOCKER_USERNAME`` and ``DOCKER_TOKEN``.
-``DOCKER_USERNAME`` should be set to the DockerHub username of the account that will be pushing the new Docker images.
+
+If you use ghcr.io as your repository (which is the FastAPI template default) you require a repository secret named ``GITHUB_TOKEN``.
```

Comment:
Note that `GITHUB_TOKEN` is built into GitHub Actions (i.e. you don't configure it, it just exists for every workflow). 

So something along the lines of:

> If you use ghcr.io as your repository (which is the FastAPI template default) you can use GitHub's built-in ``GITHUB_TOKEN``, you don't need to add an additional secret.

Reply:
Done.